build.gradle

```diff
@@ -22,9 +22,30 @@ repositories {
     mavenCentral()
 }
 
+dependencyManagement {
+    imports {
+        mavenBom 'com.linecorp.armeria:armeria-bom:0.79.0'
+    }
+}
+
 dependencies {
-    implementation 'org.springframework.boot:spring-boot-starter-web'
+    // Armeria
+    compile 'com.linecorp.armeria:armeria-spring-boot-starter'
+
```

Comment:
Does armeria have full compatibility with normal pivotal spring boot(or pivotal product family)? It seems the version naming and way of version control of the package are little bit different from original spring?

Reply:
Yes. Armeria have full compatibility with spring boot. As you concerned, Armeria would be released for every 2 weeks as consider spring boot compatibility, so it is okay.